docs/Inputs.md

```diff
@@ -1285,9 +1285,8 @@ import TextField from '@material-ui/core/TextField';
 import { Field } from 'redux-form';
 const renderTextField = ({ input, label, meta: { touched, error }, ...custom }) => (
     <TextField
-        hintText={label}
-        floatingLabelText={label}
-        errorText={touched && error}
+        helperText={label}
```

Comment:
We should put the `label` here but the `error` (which is a string) if `touched` is `true`